src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java

```diff
@@ -337,6 +337,8 @@ public void testChangeInConfig() throws Exception {
      * @noinspection ResultOfMethodCallIgnored
      * @noinspectionreason ResultOfMethodCallIgnored - temporary suppression until #11589
      */
+    // Until #11589
+    @SuppressWarnings("CheckReturnValue")
```

Comment:
this will be first tool that we allow "in-code and by-code" suppression 
is there any benefit to use https://errorprone.info/docs/installation#command-line version to have options to control its behavior ? or we can have it as maven integration for now and consider moving it out as it become problematic.
Please share your thoughts.

Reply:
I don't see any reason for not using maven, we can achieve everything by using https://errorprone.info/docs/flags, invoking it via the command line will become very tedious very quickly, so I vote for sticking to maven.

Reply:
Let’s introduce tool to via maven, we can come back to consider some system like we have for pitest as annotations become a nuisance.

Reply:
Ok